core/session.go

```diff
@@ -201,6 +204,12 @@ func (s *Session) Initialize(v *viper.Viper, scanType string) {
 	s.InitThreads()
 	s.InitAPIClient()
 
+	// check output options
+  if ((s.CSV == true && s.JSON == true) || (s.CSV == false && s.JSON == false)) && len(s.OutputFile) == 0 {
```

Comment:
@funkwhatyouheard No idea if this can be accomplished easier. I would also think since we are using string, forgo `len` and just check for `!= ""`. If the output file is `== ""` and a bool is set, just use the default output file. Don't make me give an output file, assume if I want a specific name I will give it. Set a default filename of *wraith-* and a timestamp and let them worry about it. See above comments.